### PR TITLE
Update dialogs according new design.

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:** The dialog were old design.

**Solution:** Change the html.

- Transferir adm.
screenshot from 2019-03-06 04-31-20
screenshot from 2019-03-06 04-31-41

- Confirmar vínculo

screenshot from 2019-03-06 06-36-57

- Rejeitar vínculo
screenshot from 2019-03-06 06-37-18
screenshot from 2019-03-06 08-55-22

screenshot from 2019-03-06 08-55-32
screenshot from 2019-03-06 09-10-45
screenshot from 2019-03-06 09-15-40

- Remover membro

Captura de tela de 2019-03-11 14-24-24
Captura de tela de 2019-03-11 14-23-56

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/invites/new_invite_page.html [hunk 1]
```diff
@@ -46,34 +46,34 @@ <h3><b>{{ newInviteCtrl.invite.suggestion_institution_name }}</b></h3>
               <!-- USER INFORMATION -->
               <div layout="column" ng-if="newInviteCtrl.isUserInfoImcomplete()"
                style="margin-top:7px;">
-                <md-input-container ng-if="newInviteCtrl.checkUserName()" class="no-margin">
+                <md-input-container ng-if="newInviteCtrl.checkUserName()" class="zero-margin">
```
Comment (ruanres) [inline frontend/invites/new_invite_page.html:49]: add a class to the parent div, then use composition to change the margin of all md-input-container. e.g:
`.some-class md-input-container {
     margin: 0;
}`

Add that to the new_invite.css file

File: frontend/requests/request_user_dialog.html [hunk 2]
```diff
@@ -1,37 +1,29 @@
-<md-dialog flex-gt-sm="{{requestCtrl.getSizeGtSmDialog()}}" flex-sm="80" flex-xs="100">
+<md-dialog flex-gt-sm="{{requestCtrl.getSizeGtSmDialog()}}" flex-sm="80" flex-xs="90">
     <md-dialog-content ng-if="!requestCtrl.isRejecting" class="md-dialog-content" layout-gt-sm="column" layout-padding>
-      <h4 style="margin: 0;">Confirmar Vínculo</h4>
-      <p style="margin: 0;">
+      <h3 style="margin: 0;">Confirmar Vínculo</h4>
```
Comment (ruanres) [inline frontend/requests/request_user_dialog.html]: starts with h3 and closes with h4

**Review by ruanres — CHANGES_REQUESTED** [review]

File: frontend/institution/transferAdminController.js [hunk 3]
```diff
@@ -20,6 +20,10 @@
             return false;
         };
 
+        transferAdminCtrl.getClass = function getClass(member){
```
Comment (JuliePessoa) [inline frontend/institution/transferAdminController.js:26]: jsdoc

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

File: frontend/invites/new_invite_page.html (hunk 1 above)
Comment (MaianaB) [reply frontend/invites/new_invite_page.html:49]: Eu movi essa classe porque pretendo reutilizar ela na que estou fazendo. Se eu fizer isso não terá como.

**Review by raonismaneoto — APPROVED** [review]

**Review by ruanres — APPROVED** [review]

**Review by JuliePessoa — APPROVED** [review]

**Pull request merged**